packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -128,7 +128,7 @@ export class DefaultCrudRepository<T extends Entity, ID>
     this.modelClass = dataSource.createModel<juggler.PersistedModelClass>(
       definition.name,
       properties,
-      Object.assign({strict: true}, definition.settings),
+      Object.assign({strict: true, strictDelete: true}, definition.settings),
```

Comment:
Hmm ... Based on the discussion we had in the issue I'm thinking it's fine to leave the code change out and just document how to set the `strictDelete` property on a model and what it means. A quick and simple change to the docs in this section (similar to the `strict` property): https://loopback.io/doc/en/lb4/Model.html#using-legacy-juggler should be enough imo. Thoughts?

Reply:
Yep, I am fine with that too. How do you do `<Model>.settings.strictDelete = false` in LB4, though?

Reply:
The same way you do it for `strict: false` in LoopBack 4. 

Here's an example:
```ts
// Use strictDelete to return 404 if a delete fails.
@model({settings: {strictDelete: true}})
class Todo extends Entity { ... }
```

And now we can leave strictDelete as default instead of making it `true`.